.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
I suggest creating a global `.gitignore`:

1. Create a file `.gituserignore`.
2. Put anything what do you want. E.g. my file:

```
 *.swo
 *.swp
 .DS_Store

 npm-debug.log

 puphpet/config-custom.yaml

 .grunt/
 .idea/
 .vagrant/

 /vendor/
 /node_modules/
 /bower_components/
```
3. Add to your `~/.gitconfig`

```
[core]
    excludesfile = ~/.gituserignore
```